docs/versioning.rst

```diff
@@ -0,0 +1,89 @@
+**********
+Versioning
+**********
+
+
+Current Status
+==============
+
+.. list-table:: Releases
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Release
+     - Supported Runtimes
+     - Minimum Trace Agent
+     - Status
+   * - 0.x
+     - CPython: 2.7, 3.5+
+     - 
+     - MAINTENANCE [2022-02-25]
+   * - 1.x
+     - CPython: 2.7, 3.5+
+     - 7.28
+     - CURRENT
+
+
+The **status** of each release determines when changes are made:
+
+MAINTENANCE
+    No longer under development.
+
+    Receives bug fixes and security fixes.
```

Comment:
```suggestion
    Supported with bug fixes and security fixes.
```

?

Reply:
Then can make the same change to CURRENT.

The reason for this is we can then make sure to mark DEVELOPMENT as being "not supported" (not sure exact wording).

what do you think?